docs/build/wasm/swanky-suite/cli.md

```diff
@@ -0,0 +1,329 @@
+
+This command will be updated to fix that, and provide more useful information.
+:::
+
+_Resources:_
+
+- [_`swanky check` command usage manual_](https://github.com/AstarNetwork/swanky-cli/tree/master/packages/cli#swanky-check)
+
+### Manage accounts
+
+Create and list accounts used for contract interaction.
+
+These are the accounts stored inside your `swanky.config.json`, so the command needs to be ran from within the project directory.
+
+During account creation you'll have an option of passing your own mnemonic, or have Swanky generate one for you by passing `-g` flag.
+
+You can also mark the account as "production" which will require you to set a password and encrypt the mnemonic.
+
+Be careful not to use a dev account on live networks, as their mnemonic is stored in plain text in the config!
+
+<Figure caption="Creating and listing accounts" src={require('../img/swanky/acc-create.png').default} width="65%" />
+
+:::tip
+Newly generated accounts that are not the preconfigured dev accounts (Alice, Bob, Charlie...) will have no funds initially, so you'll have to transfer some manually.
+:::
+
+_Resources:_
+
+- [_`swanky account` command usage manual_](https://github.com/AstarNetwork/swanky-cli/tree/master/packages/cli#swanky-account-create)
+
+### Interact with contracts
+
+`swanky contract` command offers several subcommands for different interactions with your contracts.
+
+<Figure caption="Different `contract` subcommands" src={require('../img/swanky/contract-commands.png').default} width="65%" />
+
+The command names are self explanatory, and to get more detailed information on using a specific command, you can use the help flag with it:
+
+```
+swanky contract SUB_COMMAND --help
+```
+
+#### Compile
+
+Depending on the contracts definition in `swanky.config.json`, calling `swanky contract compile CONTRACT_NAME` will run either cargo-contract or ask! compiler (via npm script).
+
+If you have multiple contracts and wish to compile them all at once, you can pass the `--all` flag instead of the contract name.
+
+Likewise, if you're compiling for production, you need to pass the `--prod` flag.
+
+<Figure caption="Compile all contracts" src={require('../img/swanky/compile.png').default} width="65%"/>
+
+_Resources:_
+
+- [_`swanky account` command usage manual_](https://github.com/AstarNetwork/swanky-cli/tree/master/packages/cli#swanky-contract-compile-contractname)
+
+#### Get detailed contract description
+
+Compiling the contract will generate it's metadata too.
+
+Swanky provides `contract explain CONTRACT_NAME` command to get a more human friendly version of that metadata:
+
+<Figure caption="Getting contract metadata information" src={require('../img/swanky/contract-explain.png').default} width="65%"/>
+
+_Resources:_
+
+- [_`contract compile` command usage manual_](https://github.com/AstarNetwork/swanky-cli/tree/master/packages/cli#swanky-contract-compile-contractname)
+
+#### Run E2E tests
+
```

Comment:
Maybe add a note that these are not the same as Ink! E2E tests, describe how they work in the background (in one sentence)